pythran/pythonic/__builtin__/dict/get.hpp

```diff
@@ -4,7 +4,7 @@
 #include "pythonic/include/__builtin__/dict/get.hpp"
 
 #include "pythonic/types/dict.hpp"
-#include "pythonic/types/none.hpp"
+#include "pythonic/types/NoneType.hpp"
```

Comment:
Because Pythran generate includes with ```pythonic/types/type(input_type).h```
Here , input_type for None is NoneType. I prefer have the same naming convention for every type than a special handling for None.